utils/fetch.go

```diff
@@ -44,5 +44,5 @@ func Fetch(url string, fileName string) error {
 		return err
 	}
 
-	return nil
+	return file.Sync()
```

Comment:
can you explain what this does?

Reply:
@uzaxirr recommended this approach or returning file.Close() approach in issue #42. I googled and found out file.Sync() will guarantee that the data is on disk and persistent even if the system is powered down or the operating system crashes. It is recommended to use file.Sync() for critical read/write operations. More on this can be found 
[here](https://stackoverflow.com/questions/10862375/when-to-flush-a-file-in-go) @abhishekraj272